tests/testthat/test-options.R

```diff
@@ -23,20 +21,23 @@ test_that("bcdc_options() returns a tibble",{
 test_that("bcdata.chunk_limit",{
   skip_if_net_down()
   skip_on_cran()
-  withr::local_options(list(bcdata.chunk_limit = 10000))
-  expect_error(check_chunk_limit())
+  withr::with_options(list(bcdata.chunk_limit = 100000), {
+    expect_error(check_chunk_limit())
+  })
+  withr::with_options(list(bcdata.chunk_limit = 10), {
+    expect_silent(check_chunk_limit())
+    expect_equal(check_chunk_limit(), 10)
+  })
 })
 
-test_that("bcdata.single_download_limit", {
+test_that("bcdata.single_download_limit is deprecated but works", {
```

Comment:
Maybe a comment in here to say that this can be safely removed once the option is fully deprecated? May save some future time.